docker/docker-compose.yml

```diff
@@ -10,8 +10,17 @@ services:
     entrypoint: ["/bin/sh","-c"]
     command:
       - |
-         ./aleph-node purge-chain --chain testnet1 --base-path /tmp/Damian -y
-         ./aleph-node --validator --chain testnet1 --base-path /tmp/Damian --name Damian --node-key-file /tmp/Damian/libp2p_secret --rpc-port 9933 --ws-port 9943 --port 30333 --rpc-cors all --rpc-methods Safe --execution Native --no-prometheus --no-telemetry --session-period 5 --millisecs-per-block 1000 --no-mdns
+         ./aleph-node purge-chain --chain /tmp/chainspec.json --base-path /tmp/Damian -y
```

Comment:
perhaps you can put this repeated code in some shell script included in the docker image and just call it from here?

Reply:
good point, but waiting for a new image from @bartoszjedrzejewski, that will likely have an entrypoint